docs/upgrading.md

```diff
@@ -0,0 +1,73 @@
+# Upgrading the Agent
+
+This document provides user guidance around agent upgrades and versioning.
+Like every other piece of software included in your project, the agent 
+should be upgraded frequently enough to pick up ongoing enhancements 
+while allowing room for testing. 
+
+## Watching For New Releases
+
+How can you find out about new releases? The best way is to "Watch" this repo
+from the "Watch" button in the upper-right corner of the main repository 
+page. If you don't want to be notified for every pull request and issue, you can 
+choose "Custom" under "Watch" and select just the "Releases" checkbox.
+
+Another approach is to just check in regularly with the GitHub 
```

Comment:
Do you need to provide two different ways to check the release status? If you could only suggest one, which would it be? Although you might see many different options for getting releases, your customers probably don't need more than one. Describing more than one makes the documentation harder to read, and you always have the risk of confusing the customer.

Reply:
Great point. I'll remove the second option and just link to the release page.